src/blog/2023/07/flowforge-1-9-release.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers API Documentation with Swagger UI
+subtitle: FlowForge 1.9 adds new features to make it easier to administer FlowForge
+description: The new release of FlowForge 1.9 adds new API Swagger documentation and the ability to customize Node-RED pallettes.
+date: 2023-07-06 
+authors: ["ian-skerrett"]
+image: 
+tags:
+    - posts
+    - flowforge
+    - releases
+---
+
+FlowForge 1.9 adds new features to make it easier to adminster FlowForge platform deployments, including new API documentation and the ability to create customized Node-RED palettes.
+
+<!--more-->
+
+## API Documentation
+FlowForge API allows developers to progammatically interact with the FlowForge platform. This makes it possible to integrate FlowForge into different infrastrcutre technologies, create scripts to automate specifc FlowForge tasks and embedd FlowForge into other applications.  
+
+In the 1.9 release we are now publishing our [API documentation](https://flowforge.com/docs/api/) using the [OpenAPI specification](https://swagger.io/specification/) and making it viewable with the Swagger UI. Both these industrial standards will make using the FlowForge API easier to use and understand.
+
+
+## Customize Node-RED Palettes  [#2002](https://github.com/flowforge/flowforge/issues/2002)
+
+FlowForge platform adminstrators are now able to create customized Node-RED palettes that will be used when a Node-RED instance is created. An adminstator can create pre-defined templates to specify the nodes that should be included in the palette. This makes it easier for FlowForge teams to standardized on Node-RED usage across an organization.
```

Comment:
It's not for teams, so the last sentence isn't correct? This is only useful on-prem?

Reply:
See my suggestion above